src/flag_gems/runtime/register.py

```diff
@@ -1,37 +1,39 @@
-import torch
-
 from . import backend, commom_utils, error
 from .backend.device import DeviceDetector
 
-aten_lib = torch.library.Library("aten", "IMPL")
-
 
 class Register:
     def __init__(
         self,
         config,
         user_unused_ops_list=None,
         lib=None,
-        debug=False,
+        debug=True,
     ):
         # lib is a instance of torch.library.Library
-        self.lib = lib
         self.device = DeviceDetector()
+        self.vendor_list = list(commom_utils.vendors_map)
+        self.check_backend()
```

Comment:
I don't think it's necessary. as long as Register successfully initializes self.device, backend won't be wrong.

Reply:
double check, Prevents user-transmitted vendor errors

Reply:
isn't vendor detected from backend? why permit user to transmit it?